### PR TITLE
Spanish blueprint doc

### PR DESCRIPTION
**Pull request by lfiedlerc**
Documentation for the Spanish screening app.

Note that an image with the file structure is missing and will have to be generated on a Mac.

### CODE REVIEW

File: source/blueprints/screening_app.rst [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+   conv.say('quiero conocer mi riesgo')
+
+.. code-block:: console
+
+   ['¿Cuál es su edad?', 'Listening...']
+
+The :meth:`say` method:
+
+ - packages the input text in a user request object
+ - passes the object to the MindMeld Application Manager to a simulate an external user interaction with the app, and
```
Comment (serapio) [inline source/blueprints/screening_app.rst:445]:
```suggestion
 - passes the object to the MindMeld Application Manager to simulate an external user interaction with the app, and
```

File: source/blueprints/screening_app.rst [hunk 2]
```diff
@@ -0,0 +1,485 @@
+Health Screening
+================
+
+In this step-by-step walkthrough, you'll build a conversational application for a health screening that allows users to identify their risk of prediabetes.
+
+Working through this blueprint will teach you how to
+
+   - configure a MindMeld application in a language different than English
+   - handle a large number of domains and intents
+   - learn non-system entities
```
Comment (serapio) [inline source/blueprints/screening_app.rst:10]:
```suggestion
   - learn custom entities
```

**Review by serapio — CHANGES_REQUESTED** [review]

File: source/blueprints/screening_app.rst [hunk 3]
```diff
@@ -0,0 +1,507 @@
+Health Screening
+================
+
+In this step-by-step walkthrough, you'll build a conversational application for a health screening that allows users to identify their risk of prediabetes.
```
Comment (vembar) [inline source/blueprints/screening_app.rst:4]: "...for a health screening that..." --> "...for a health screening application that.."

File: source/blueprints/screening_app.rst [hunk 4]
```diff
@@ -0,0 +1,492 @@
...
+
+Working through this blueprint will teach you how to
+
+   - configure a MindMeld application in a language different than English
+   - learn custom entities
+   - use dialogue flows to structure the conversation
+
+.. note::
+
+   Please make sure to install and run all of the :ref:`pre-requisites <getting_started_virtualenv_setup>` for MindMeld before continuing on with this blueprint tutorial.
```
Comment (ritvikshrivastava) [inline source/blueprints/screening_app.rst:14]:
```suggestion
   Please make sure to install and run all of the :ref:`pre-requisites <getting_started_virtualenv_setup>` for MindMeld before continuing with this blueprint tutorial.
```

File: source/blueprints/screening_app.rst [hunk 5]
```diff
@@ -0,0 +1,492 @@
...
+^^^^^^^^^^^^^^^
+
+This screening application would be offered by a public health department through a messaging service like SMS or WhatsApp. Users should be able to provide answers to a series of questions asked in Spanish and receive an assessment about their risk for prediabetes. They should be able to opt-out of the questionnaire at any time.
+A user's risk is calculated based on the `questionnaire <https://www.cdc.gov/diabetes/risktest/index.html>`_ provided by the American Diabetes Association and the Centers for Disease Control and Prevention.
+
+
+2. Example Dialogue Interactions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The conversational flows for a health screening will primarily involve asking users a series of questions about their medical history relevant to prediabetes. Once all necessary questions have been asked, the users are informed of their risk for presenting prediabetes.
```
Comment (ritvikshrivastava) [inline source/blueprints/screening_app.rst:26]:
```suggestion
The conversational flows for a health screening will primarily involve asking users a series of questions about their medical history relevant to prediabetes. Once all necessary questions have been answered, the users are informed of their risk for presenting prediabetes.
```

File: source/blueprints/screening_app.rst [hunk 6]
```diff
@@ -0,0 +1,507 @@
...
```
Comment (vembar) [inline source/blueprints/screening_app.rst:26]: The conversational flows for ~~a~~ health screening will primarily ...

File: source/blueprints/screening_app.rst [hunk 7]
```diff
@@ -0,0 +1,492 @@
...
+   @screen_prediabetes.handle(intent='answer_yes')
+   def confirm_send_next(request, responder):
+      ...
+      
+   @app.handle(intent='answer_no')
+   @screen_prediabetes.handle(intent='answer_no')
+   def negate_send_next(request, responder):
+      ...
+
+The ``@app.dialogue_flow`` decorator designates the flow's entry point. Once inside, every follow up turn will continue to be in this flow until we exit the flow. If the user tries to disrupt the flow by answering something unrelated to the screening, we gently reprompt the user to provide an answer for the current question. We can further control how the screening progresses by setting the ``allowed_intents`` attribute of the params object in the dialogue state handler. This attribute specifies a list of intents that you can set to force the language processor to choose from.
```
Comment (ritvikshrivastava) [inline source/blueprints/screening_app.rst:178]:
```suggestion
The ``@app.dialogue_flow`` decorator designates the flow's entry point. Once inside, every follow up turn will continue to be in this flow until we exit the flow. If the user response disrupts the flow by answering something unrelated to the prompt, we gently reprompt the user to provide an answer for the current question. We can further control how the screening progresses by setting the ``allowed_intents`` attribute of the params object in the dialogue state handler. This attribute specifies a list of intents that you can set to force the language processor to choose from.
```

File: source/blueprints/screening_app.rst [hunk 8]
```diff
@@ -0,0 +1,507 @@
...
+    Best accuracy: 96.90%, params: {'class_weight': {0: 1.0209401709401709, 1: 1.006530612244898, 2: 0.7894047619047618, 3: 1.0645631067961165, 4: 1.268939393939394, 5: 1.07363184079602, 6: 1.4362745098039216, 7: 0.8722477064220183, 8: 0.9802238805970149, 9: 2.202, 10: 1.2960317460317459, 11: 1.1579268292682925}}
+
+You can use similar options to inspect and experiment with the Entity Recognizer and the other NLP classifiers. Finding the optimal machine learning settings is a highly iterative process involving several rounds of model training (with varying configurations), testing, and error analysis. See the :doc:`User Guide <../userguide/nlp>` for more about training, tuning, and evaluating the various MindMeld classifiers.
+
+.. admonition:: Exercise
+
+   Experiment with different models, features, and hyperparameter selection settings to see how they affect classifier performance. Maintain a held-out validation set to evaluate your trained NLP models and analyze misclassified test instances. Then, use observations from the error analysis to inform your machine learning experimentation. See the :doc:`User Guide <../userguide/nlp>` for examples and discussion.
```
Comment (vembar) [inline source/blueprints/screening_app.rst:416]: "Maintain a held-out validation set to evaluate your trained NLP models and analyze misclassified ~~test~~ instances."

Just so we are clear that we only analyze errors on the validation set and not the test set.

File: source/blueprints/screening_app.rst [hunk 9]
```diff
@@ -0,0 +1,507 @@
...
+   - ``answer_yes_gestational`` — User wants to answer the question regarding gestational diabetes with an implied "yes". For example, the answer: "During my first pregnancy" implies an affirmative response.
+   - ``answer_yes_family`` — User wants to answer the question regarding family history of diabetes with an implied "yes".
+   - ``answer_yes_hbp`` — User wants to answer the question regarding high blood pressure with an implied "yes".
+   - ``answer_yes_active`` — User wants to answer the question regarding physical activity with an implied "yes".
+   - ``answer_height`` — User wants to provide their height.
+   - ``answer_weight`` — User wants to provide their weight.
+
+There are two types of entities in MindMeld: :ref:`System Entities <system-entities>` and :doc:`Custom Entities <../userguide/entity_recognizer>`. System entities are pre-defined in MindMeld. Examples include ``sys_temperature``, ``sys_time``, and ``sys_interval``. Custom entities are defined by the developers of each application. Within each entity folder, the file ``gazetteer.txt`` contains the full list of values for each custom entity.
```
Comment (vembar) [inline source/blueprints/screening_app.rst:66]: Should we clarify that the gazetteer offers a strong signal and does not mean we can identify only those entity values?

**Review by serapio — APPROVED** [review]
